docs/data/pages.ts

```diff
@@ -23,144 +23,171 @@ const pages: MuiPage[] = [
-          { pathname: '/x/react-data-grid/row-updates' },
-          { pathname: '/x/react-data-grid/row-height' },
-          { pathname: '/x/react-data-grid/row-spanning', newFeature: true },
-          { pathname: '/x/react-data-grid/master-detail', plan: 'pro' },
-          { pathname: '/x/react-data-grid/row-ordering', plan: 'pro' },
-          { pathname: '/x/react-data-grid/row-pinning', plan: 'pro' },
-          { pathname: '/x/react-data-grid/row-recipes', title: 'Recipes' },
+          { pathname: '/x/react-data-grid/tree-data', plan: 'pro' },
+          { pathname: '/x/react-data-grid/row-grouping', plan: 'premium' },
+          { pathname: '/x/react-data-grid/aggregation', plan: 'premium' },
+          { pathname: '/x/react-data-grid/pivoting', plan: 'premium', planned: true },
         ],
       },
-      { pathname: '/x/react-data-grid/editing' },
-      { pathname: '/x/react-data-grid/sorting' },
       {
-        pathname: '/x/react-data-grid/filtering-group',
-        title: 'Filtering',
+        pathname: '/x/react-data-grid/visual-customization-group',
+        subheader: 'Visual customization',
         children: [
-          { pathname: '/x/react-data-grid/filtering', title: 'Overview' },
-          { pathname: '/x/react-data-grid/filtering/customization' },
-          { pathname: '/x/react-data-grid/filtering/quick-filter' },
-          { pathname: '/x/react-data-grid/filtering/server-side', title: 'Server-side filter' },
-          { pathname: '/x/react-data-grid/filtering/multi-filters', plan: 'pro' },
-          {
-            pathname: '/x/react-data-grid/filtering/header-filters',
-            plan: 'pro',
-            newFeature: true,
-          },
-          { pathname: '/x/react-data-grid/filtering-recipes', title: 'Recipes' },
+          { pathname: '/x/react-data-grid/style', title: 'Styling basics' },
+          { pathname: '/x/react-data-grid/style-recipes', title: 'Styling recipes' },
+          { pathname: '/x/react-data-grid/layout' },
+          { pathname: '/x/react-data-grid/overlays', title: 'Overlays' },
+          { pathname: '/x/react-data-grid/components', title: 'Custom subcomponents' },
         ],
       },
-      { pathname: '/x/react-data-grid/pagination' },
       {
-        pathname: '/x/react-data-grid/selection',
+        pathname: '/x/react-data-grid/interactivity-group',
+        subheader: 'Interactivity',
         children: [
-          { pathname: '/x/react-data-grid/row-selection' },
-          { pathname: '/x/react-data-grid/cell-selection', plan: 'premium', newFeature: true },
+          { pathname: '/x/react-data-grid/localization' },
+          {
+            pathname: '/x/react-data-grid/custom-behavior',
+            children: [
+              { pathname: '/x/react-data-grid/api-object', title: 'API object' },
+              { pathname: '/x/react-data-grid/events' },
+              { pathname: '/x/react-data-grid/state' },
+            ],
```

Comment:
These pages are quite important for advanced use cases - I feel like they would benefit from being visible at the top level and not nested within a group. But I'm not certain that they all belong under **Interactivity**, either.

Reply:
Yeah, I think the grouping might've made more sense before introducing these new subheaders, so maybe not necessary anymore. I could drop the group and move them directly under "Advanced features."

Reply:
How about moving them to the Resources section?